src/jace/api.py

```diff
@@ -72,18 +75,19 @@ def jit(
         fun: Function to wrap.
         primitive_translators: Use these primitive translators for the lowering to SDFG.
             If not specified the translators in the global registry are used.
-        kwargs: Jit arguments.
+        kwargs: Jit arguments, see `JITOptions` for more.
 
     Note:
         This function is the only valid way to obtain a JaCe computation.
     """
-    if kwargs:
+    not_supported_jit_keys = kwargs.keys() - {"backend"}
+    if not_supported_jit_keys:
         # TODO(phimuell): Add proper name verification and exception type.
         raise NotImplementedError(
-            f"The following arguments to 'jace.jit' are not yet supported: {', '.join(kwargs)}."
+            f"The following arguments to 'jace.jit' are not yet supported: {', '.join(not_supported_jit_keys)}."
```

Comment:
I would also raise a `RuntimeError` or a `ValueError`
```suggestion
            f"The following arguments to 'jace.jit' are not supported: {', '.join(not_supported_jit_keys)}."
```